src/iotjs_binding.cpp

```diff
@@ -256,7 +256,6 @@ JObject JObject::GetProperty(const char* name) {
       _obj_val.u.v_object,
       reinterpret_cast<const jerry_api_char_t*>(name),
       &res);
-  IOTJS_ASSERT(is_ok);
```

Comment:
This caused me a headache too. There where a bug in the Jerry API: https://github.com/Samsung/jerryscript/pull/1042

Currently the assert fails on `iotjs_module_tcp.cpp:86`

`JObject::GetProperty` returns with the result of the `jerry_api_get_object_field_value` call. Every GetProperty() checks the return value in runtime, so they are expecting that the property might not exist. There is no description for `JObject::GetProperty`. If we leave the assert there, then you can call the `GetProperty` with existing property names. IMHO, this is a too strict limit.